solvers/amigo.py

```diff
@@ -20,13 +20,47 @@ class Solver(StochasticJaxSolver):
     Bilevel Optimization". ICLR 2022"""
     name = 'AmIGO'
 
+    """How to add a new stochastic solver to the benchmark?
```

Comment:
Put this in a `template_stochastic_solver`, with a simpler algorithm without subroutines (SOBA?)

Add some explanation why we do this? (jittting for fast stochastic iterations)